PWGHF/HFJ/Tasks/taskDstarPolarisationInJet.cxx

```diff
@@ -0,0 +1,917 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file taskDstarPolarisationInJet.cxx
+/// \brief Analysis task for Dstar spin alignment in jet (rho00 vs. z||)
+///
+/// \author F. Grosa (CERN) fabrizio.grosa@cern.ch
+/// \author Mingze Li (CCNU) Mingze.li@cern.ch
+
+#include "PWGHF/Core/DecayChannels.h"
+#include "PWGJE/Core/FastJetUtilities.h"
+#include "PWGJE/Core/JetFinder.h"
+#include "PWGJE/Core/JetHFUtilities.h"
+#include "PWGJE/Core/JetUtilities.h"
+#include "PWGJE/DataModel/Jet.h"
+
+#include <Framework/AnalysisTask.h>
+#include <Framework/HistogramRegistry.h>
+#include <Framework/runDataProcessing.h>
+
+#include <Math/GenVector/Boost.h>
+#include <Math/Vector3D.h>
+#include <Math/Vector4D.h>
+#include <TRandom3.h>
+#include <TVector3.h>
```

Comment:
These are not used anywhere.
Please apply IWYU.

Reply:
OK, actually:
`ROOT::Math::Boost` is provided by `Math/GenVector/Boost.h`.
`PxPyPzMVector` is provided by `Math/Vector4D.h`.
Thanks for the clarification.